### PR TITLE
License fixed

### LINKED ISSUE #98: Fixing the Licence file

### PR DESCRIPTION
**Pull request by atabak-alishiri**
added the creative license and also the line about the dataset. ready to merge

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
-MIT License
+# Instructional Material
```
Comment (farrandi) [inline LICENSE.md:1]: Is this the correct heading? (I really do not know)

**Comment by salva-u** [comment]
the license update in readme will be done by nando ? just double checking

**Comment by salva-u** [comment]
okay just checked the issues part, I think @atabak-alishiri you can add it to readme similar to other people's repo , the license of the dataset mention

**Comment by rbouwer** [comment]
Same comment as @salva-u but otherwise looks good to me!

**Comment by atabak-alishiri** [comment]
The readme has been updated now, thanks for the feedback.

**Review by rbouwer — APPROVED** [review]
LGTM!

**Review by salva-u — APPROVED** [review]

**Review by farrandi — APPROVED** [review]

**Pull request merged**